main_test.go

```diff
@@ -152,6 +152,31 @@ func TestTransferTokensToAccountActions(t *testing.T) {
 
 		// TODO: Assert that the NFT has been received by the recipient account
 	})
+
+	t.Run("Signers shouldn't be able to transfer 0.0 fungible tokens out of the Treasury", func(t *testing.T) {
+		otu.ProposeFungibleTokenTransferAction("treasuryOwner", Signers[0], RecipientAcct, 0.0)
```

Comment:
will need to update the tests here to make sure you can't propose a transfer of 0.0 tokens